lib/logger/once.ts

```diff
@@ -12,7 +12,8 @@ type OmitFn = (...args: any[]) => any;
  */
 function getCallSite(omitFn: OmitFn): string | null {
   const stackTraceLimitOrig = Error.stackTraceLimit;
-  const prepareStackTraceOrig = Error.prepareStackTrace;
+  // Keep a safe copy of the original implementation
+  const prepareStackTraceOrig = Error.prepareStackTrace.bind(Error);
```

Comment:
I don't think this error is _new_, I think that it's maybe been surfaced due to .cache busting

Reply:
This changes nothing, but makes the variable any. We simply set it back below (line 33-34/ 34/35)

Reply:
eslint warning should be ignored here